x/ibc-hooks/contracts/crosschain-swaps/src/consts.rs

```diff
@@ -0,0 +1,9 @@
+// Msg Reply IDs
+pub const SWAP_REPLY_ID: u64 = 1u64;
+pub const FORWARD_REPLY_ID: u64 = 2u64;
+
+// IBC timeout
+pub const PACKET_LIFETIME: u64 = 86400u64;
```

Comment:
Can we comment what this is / what units they are?

I assume seconds / 24 hours?

May want to consider elongating this

Reply:
what value would you suggest? a week?

Reply:
yeah a week timeout sgtm

Reply:
changed